src/ploneorg/addonlisting/browser/views.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from Products.Five.browser import BrowserView
-from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
-
+from plone.protect.interfaces import IDisableCSRFProtection
 from ploneorg.addonlisting.utils import update_addon
 from ploneorg.addonlisting.utils import update_addon_list
-from plone.protect.interfaces import IDisableCSRFProtection
```

Comment:
yes should be removed later, but now I just do add automated tests and make the tests green for flake8 and isort errors.

@pavithirakc that is in there for first debug in the browser views. It should be removed in the final product.